packages/wonder-blocks-form/src/components/text-field.tsx

```diff
@@ -327,81 +335,35 @@ const styles = StyleSheet.create({
             color: color.offBlack64,
         },
     },
-    defaultFocus: {
-        ":focus-visible": {
-            borderColor: color.blue,
-            outline: `1px solid ${color.blue}`,
-            outlineOffset: 0, // Explicitly set outline offset to 0 because Safari sets a default offset
-        },
-    },
     error: {
         background: color.fadedRed8,
         border: `1px solid ${color.red}`,
         color: color.offBlack,
         "::placeholder": {
             color: color.offBlack64,
         },
-        ":focus-visible": {
-            outlineColor: color.red,
-            borderColor: color.red,
-        },
     },
     disabled: {
         background: color.offWhite,
         border: `1px solid ${color.offBlack16}`,
         color: color.offBlack64,
         "::placeholder": {
-            color: color.offBlack64,
-        },
-        cursor: "not-allowed",
-        ":focus-visible": {
-            outline: "none",
-            boxShadow: `0 0 0 1px ${color.white}, 0 0 0 3px ${color.offBlack32}`,
+            color: color.offBlack32,
```

Comment:
**question:** Is this going to be modified in the future PR? See related comment here: https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=66cf79973b48a29341523038

Reply:
Yes that's right! I'll put all these changes back in the future PR and then fix the outline styles so it can work in all cases. These changes will just revert the TextField styles to what they were before!

Reply:
got it, sounds good, thanks for the clarification :)